mcs/class/System/System.Net.Configuration/DefaultProxySection.cs

```diff
@@ -63,6 +63,7 @@ static DefaultProxySection ()
 			properties.Add (bypassListProp);
 			properties.Add (moduleProp);
 			properties.Add (proxyProp);
+			properties.Add (useDefaultCredentialsProp);
```

Comment:
Looks like we're missing `enabledProp` here too. I'll do a pass through the config sections since at a quick glance at least one other class has the same issue :)